app/controllers/posts_controller.rb

```diff
@@ -18,53 +18,48 @@ def add_profile_username
     end
     if @profile == nil
       respond_to do |format|
-        format.html { redirect_to council_posts_path(@council), flash: {alert: 'Hittade ingen användare med det användarnamnet.'}}
+        format.html { redirect_to council_posts_path(@council), flash: {alert: 'Hittade ingen användare med det användarnamnet.'} }
       end
-    elsif @profile.name.blank?          
-      redirect_to council_posts_path(@council), flash: {alert: 'Användaren :"' + @user.username.to_s + '" måste fylla i fler uppgifter i sin profil.'} 
+    elsif @profile.name.blank?
+      redirect_to council_posts_path(@council), flash: {alert: 'Användaren :"' + @user.username.to_s + '" måste fylla i fler uppgifter i sin profil.'}
     elsif @profile.posts.include?(@post)
-      redirect_to council_posts_path(@council), flash: {alert: @profile.name.to_s + '(' + @user.username.to_s + ') har redan posten '+@post.title.to_s + '.'}         
+      redirect_to council_posts_path(@council), flash: {alert: @profile.name.to_s + '(' + @user.username.to_s + ') har redan posten '+@post.title.to_s + '.'}
```

Comment:
Line is too long. [157/100]<br>Surrounding space missing for operator '+'.<br>Space inside { missing.<br>Space inside } missing.